torch/_torch_docs.py

```diff
@@ -619,6 +619,105 @@ def merge_dicts(*dicts):
     True
 """)
 
+add_docstr(torch.all,
+           r"""
+all() -> BoolTensor
+
+Returns True if all elements in the tensor are True, False otherwise.
+
+Example::
+
+    >>> a = torch.rand(1, 2).bool()
+    >>> a
+    tensor([[False, True]], dtype=torch.bool)
+    >>> torch.all(a)
+    tensor(False, dtype=torch.bool)
+    >>> a = torch.arange(0, 3)
+    >>> a
+    tensor([0, 1, 2])
+    >>> torch.all(a)
+    tensor(False)
+
+.. function:: all(input, dim, keepdim=False, out=None) -> Tensor
+
+Returns True if all elements in each row of the tensor in the given
+dimension :attr:`dim` are True, False otherwise.
```

Comment:
It's ambiguous whether it returns True for each row, or a single True value for all rows. 
Maybe something like "Reduces each row of :attr:\`input\` in the given dimension :attr:\`dim\` to `True` if all elements in the row evaluate to \`True\`, \`False\` otherwise."